src/language/json/monaco.contribution.ts

```diff
@@ -4,7 +4,8 @@
  *--------------------------------------------------------------------------------------------*/
 
 import * as mode from './jsonMode';
-import { Emitter, IEvent, languages } from '../../fillers/monaco-editor-core';
+import { Emitter, IEvent, languages, Uri } from '../../fillers/monaco-editor-core';
+import type { JSONWorker } from './jsonWorker';
```

Comment:
We want that all exported types are in monaco.contribution.ts. So also a interface describing the  `JSONWorker` should be defined here.
Can you add a IJSONWoker. I suggest it is limited to the function you want to expose.